adafruit_atecc/adafruit_atecc.py

```diff
@@ -52,8 +52,20 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ATECC.git"
 
+
+def _convert_i2c_addr_to_atecc_addr(i2c_addr=0x60):
+    int(i2c_addr, 16)
+    return i2c_addr << 1
+
+
 # Device Address
-_REG_ATECC_ADDR = const(0xC0)
+_I2C_ADDR = 0x60
+_REG_ATECC_ADDR = _convert_i2c_addr_to_atecc_addr(i2c_addr=_I2C_ADDR)
+
+# TODO: Verify that _REG_ATECC_ADDR is still 0xC0
```

Comment:
are these TODO's supposed to still be here?

Reply:
I've added them just for the review in order to give some pointers of what to verify.
For example the different representation of the configuration bytes look quite drastic and I wanted to show that I just modify the I2C configuration in byte 16. As the config can only be applied once it should be clear that the changes will do no harm.

Reply:
ok!